presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Standard.java

```diff
@@ -79,6 +81,8 @@ public Standard(
         this.dockerFiles = requireNonNull(dockerFiles, "dockerFiles is null");
         this.portBinder = requireNonNull(portBinder, "portBinder is null");
         this.imagesVersion = requireNonNull(environmentConfig, "environmentConfig is null").getImagesVersion();
+        this.prestoBaseImage = environmentConfig.getPrestoBaseImage();
```

Comment:
i think that if this anyhow prepares us to https://github.com/prestosql/presto/pull/5508, we would need two separate environment configs. It seems to me that hadoop flavor and Java version for Presto are independent dimensions.

Reply:
Hadoop flavor is already separate configuration property